README.md

```diff
@@ -57,4 +57,10 @@ password: adminpassword
 role: ADMIN
 ```
 
-You should be able to create more users by running the app and visiting `http://localhost:8080/register` or adding them straight to your database.
\ No newline at end of file
+You should be able to create more users by running the app and visiting `http://localhost:8080/register` or adding them straight to your database.
+
+***
+
+## Style Guide
+
+https://www.oracle.com/technetwork/java/codeconventions-150003.pdf
```

Comment:
I've actually set up spotless to follow the google style guide - https://google.github.io/styleguide/javaguide.html

Reply:
Changed to Google